internal/cli/shim.go

```
@@ -190,16 +192,27 @@
+
 	cmdShape := exec.Cmd{
 		Path:   proc.Command,
 		Args:   append([]string{proc.Command}, proc.Args...),
 		Dir:    proc.Cwd,
 		Env:    env,
-		Stdin:  os.Stdin,
-		Stdout: outw,
-		Stderr: errw,
+		Stdin:  tty,
+		Stdout: tty,
+		Stderr: errw, // TODO: should it go into tty???
```

golangci-lint (1.62.2)

internal/cli/shim.go:211: Line contains TODO/BUG/FIXME: "TODO: should it go into tty???"

(godox)